### PR TITLE
Belt for botanists.

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Adds a belt for botanists can can hold sprays, bottles, beakers, syringes, wrenches, hatchets, spades, cultivators, plant analyzers, botanist gloves, floral somatorays, seeds, lighters, rolling papers, cigs and plant gene disks.

## Why It's Good For The Game
Less inventory clutter for Botany players is nice.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

image

image

image

image

Belt icon : 

image

Belt icons : 

image

Belt with all icon overlays : 

image

</details>

## Changelog
:cl:
add: Added a new Botanical belt
add: Added the botanical belt to the Botany lockers and the Biogenerator
imageadd: Added icons for the Botanical belt
imageadd: Added belt icons for the Plant Analyzer, Pest Spray, Hatchet, Spade and Cultivator
code: Replaced all can_hold lists with static lists for belts
/:cl:

### CODE REVIEW

**Review by PestoVerde322 — COMMENTED** [review]
Icon and code looks fine, do the belt need to be spawned on the maps or it has to be obtained?

**Comment by CydiaLamiales** [comment]
> Icon and code looks fine, do the belt need to be spawned on the maps or it has to be obtained?

It's like the medibelt, it spawns in the department lockers. ~~smh didn't read the changelog~~

**Review by PestoVerde322 — DISMISSED** [review]
After i got some eyedrops, it's good to go, the item spawn in the lockers.
Approved.

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -74,6 +74,40 @@
...
+	worn_icon_state = "botanical"
+	content_overlays = TRUE
+
+/obj/item/storage/belt/botanical/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_items = 7
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.max_combined_w_class = 21
+	STR.can_hold = typecacheof(list(
```
Comment (Archanial) [inline code/game/objects/items/storage/belt.dm]: make this static

**Review by Archanial — CHANGES_REQUESTED** [review]

**Review by Archanial — APPROVED** [review]

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/game/objects/items/storage/belt.dm]: Done

**Pull request merged**